doc/man1/openssl-cmp.pod.in

```diff
@@ -698,22 +698,24 @@ If not given here, the password will be prompted for if needed.
 
 For more information about the format of B<arg> see the
 B<PASS PHRASE ARGUMENTS> section in L<openssl(1)>.
-
-=item B<-engine> I<id>
-
-Specifying a crypto engine B<id> will lead to obtaining a functional
-reference to the specified engine, initializing it if needed.
-The engine will be used for all algorithms supported for keys
-prefixed by C<engine:>.
-Engines may be defined in the OpenSSL config file as usual in an engine section.
-
-Options specifying keys, like B<-key>, B<-newkey>, B<-tls_key> can prefix
-C<engine:> to engine-specific identifiers for security tokens objects held by
-the engine.
- The following example utilizes the RFC 7512 PKCS #11 URI scheme
-as supported, e.g., by libp11:
-C<-key engine:pkcs11:object=my-private-key;type=private;pin-value=1234>
-
+{- if (!$disabled{"deprecated-3.0"}) {
```

Comment:
There's otherwise an option to turn template output off and on:

```
{- output_off() if $disabled{"deprecated-3.0"}; "" -}

=item B<-engine> I<id>
...
...
C<-key engine:pkcs11:object=my-private-key;type=private;pin-value=1234>

{- output_on() if $disabled{"deprecated-3.0"}; "" -}
```

That way, you don't have to re-edit the text, just insert the `output_off` and `output_on` lines where needed.

Reply:
Okay, changed.